python/tests/test_session.py

```diff
@@ -139,10 +144,10 @@ def test_failing_case(self):
 
         session.stop(False)
 
-    def test_update_method(self):
+    def test_raises_exception_if_result_value_not_used(self):
         session = SauceSession()
 
         session.start()
 
-        session.update_test_result(True)
-        session.stop()
+        with pytest.raises(TypeError):
+            session.stop()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'result' in method call](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4711738)